temp.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Delete this

Reply:
https://github.com/kubedb/apimachinery/blob/master/apis/kubedb/v1alpha2/pgbouncer_types.go#L215
Also remove this field & run `make gen`